src/antsibull_changelog/cli.py

```diff
@@ -609,7 +614,8 @@ def command_release(args: Any) -> int:
     version, codename = _get_version_and_codename(
         paths, config, collection_details, args
     )
-
+    if args.update_galaxy_file:
+        collection_details.update_galaxy(version=version)
```

Comment:
Please keep a newline between this code and loading the changes.
```suggestion
        collection_details.update_galaxy(version=version)

```